CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
```

Comment:
Hi @thyarles 
Good morning
I found below errors on my testing
a) If we have incorrect path in wipe section then cy-runner is not able to detect
b) For some apps, we use test from cy-runner. With this PR, those paths are now been considered as invalid